saleor/payment/migrations/0041_add_calculation_transaction_events.py

```diff
@@ -0,0 +1,103 @@
+from decimal import Decimal
+
+from django.db import migrations
+
+
+def create_transaction_events(apps, schema_editor):
+    TransactionItem = apps.get_model("payment", "TransactionItem")
+    TransactionEvent = apps.get_model("payment", "TransactionEvent")
+    create_event_for_authorized(TransactionItem, TransactionEvent)
+    create_event_for_charged(TransactionItem, TransactionEvent)
+    create_event_for_refunded(TransactionItem, TransactionEvent)
+    create_event_for_canceled(TransactionItem, TransactionEvent)
+
+
+def create_event_for_authorized(TransactionItem, TransactionEvent):
+    transaction_events = []
+    authorized_transactions = TransactionItem.objects.exclude(
+        authorized_value=Decimal(0)
+    ).values_list("id", "authorized_value", "currency")
+    for pk, amount, currency in authorized_transactions:
```

Comment:
Did you check it on the big data set? It seems like it may fail.

Reply:
I doubt that any big set of the transaction already exists in a production environment. It is a preview feature, and I was consulting this migration with SRE team. The dataset that could exist on our prod  envs is around of couple thousands of records, so it will be ok. Thanks for double-checking and raising this.